package_test.go

```diff
@@ -496,6 +496,33 @@ func (s *PackageSuite) TestNulls(c *C) {
 }
 
 func (s *PackageSuite) TestValidGet(c *C) {
+	// Embedded/nested structs.
+	type E1 struct {
+		Street string `db:"street"`
+	}
+	type E2 struct {
+		ID int `db:"id"`
+	}
+	type E3 struct {
+		District string `db:"district"`
+		E2       E2
+	}
+	type EmbeddedStruct struct {
+		E1 E1
+		E3 *E3
+	}
+
+	// Struct with pointers.
+	type PointerStruct struct {
+		Street   *string         `db:"street"`
+		District *sql.NullString `db:"district"`
+		ID       *int            `db:"id"`
+	}
+
+	// Pointer variables for pointer struct test.
+	var mainStreet = "Main Street"
+	var oneThousand = 1000
```

Comment:
I notice this in existing code, so this might not be an issue for this PR, but would it be good to use `:=` here? The convention I like to use is
```go
var foo *Bar  // if the zero value of foo is needed
foo := &Bar{} // otherwise
```

Reply:
True, I should stick to this convention. I just need to remember it when I'm actually writing the code haha.